tools/actool.mk

```diff
@@ -4,6 +4,7 @@ $(call setup-stamp-file,ACTOOL_STAMP)
 BGB_STAMP := $(ACTOOL_STAMP)
 BGB_PKG_IN_REPO := Godeps/_workspace/src/github.com/appc/spec/actool
 BGB_BINARY := $(ACTOOL)
+BGB_ADDITIONAL_GO_ENV := GOARCH=$(GOARCH_FOR_BUILD) CC=$(CC_FOR_BUILD)
```

Comment:
An excerpt from src/cmd/cgo/doc.go:

> When cross-compiling, you must specify a C cross-compiler for cgo to
use.  You can do this by setting the CC_FOR_TARGET environment
variable when building the toolchain using make.bash, or by setting
the CC environment variable any time you run the go tool.  The
CXX_FOR_TARGET and CXX environment variables work in a similar way for
C++ code.

Alright then.